### ISSUE TITLE
acu is accuracy or attack success rate?

### ISSUE THREAD

**Issue by shubhamk-76** [TheMoss7/SID#2]
sum += (model(images).argmax(1) != (gt+1)).detach().sum().cpu()
!= suggest it acu is attack success rate. But acu is shorthand for accuracy so it is confusing. 
print(model_name + '  acu = {:.2%}'.format(sum / length))

Regards
Shubham Kumar

**Comment by TheMoss7** [comment]
Thank you for your correction. Indeed, acu refers to the attack success rate, and we apologize for any confusion. Our work focuses on adversarial attacks on visual classification models, so here acu actually refers to the attack success rate, and we have corrected this in the code. 
This work has already been published at ICCV 2025. Thank you for your attention.
